en/flight_controller/omnibus_f4_sd.md

```diff
@@ -222,14 +219,12 @@ make omnibus_f4sd_default
 ### Installing other firmwares
 
 If you later on want to switch back to *Betaflight*, you can easily do that:
-- Backup the parameters by copying the **params** file on the SD card (or use another SD card)
+- Backup the parameters, e.g. by [exporting](https://dev.px4.io/master/en/advanced/parameters_and_configurations.html#exporting-and-loading-parameters) them to an SD card
```

Comment:
Should this say "to an SD card"? Or should it just say "to offline storage/SD card". I'll merge this anyway.

Reply:
I think SD card is more appropriate.